frontend/src/metabase-lib/parameters/utils/parameter-type.ts

```diff
@@ -22,6 +22,11 @@ function splitType(parameterOrType: Parameter | string) {
   return parameterType.split("/");
 }
 
+export function isIdParameter(parameter: Parameter | string) {
```

Comment:
Keep the logic to check parameter type in here. I think the signature could be refactored to `isParameterType(parameter: Parameter, type: ParameterType)`, so we don't have to import separate function to determine parameter types.